experimental/oak_baremetal_launcher/src/main.rs

```diff
@@ -58,6 +58,10 @@ struct Args {
     #[clap(subcommand)]
     mode: Mode,
 
+    /// Consistent response size that the runtime should apply
+    #[clap(long, default_value = "1024")]
+    constant_response_size: u32,
```

Comment:
Does this mean we are moving away from the config file for the policy? Not that it matters too much at this point, since this is the only value that is really part of the policy anyways.